src/index.ts

```diff
@@ -1,4 +1,4 @@
-import addStyles from './add-styles';
+import addStyles from "./add-styles";
```

Comment:
I think this got converted to double quotes somehow. If you'd revert, that'd be appreciated.